dom.bs

```diff
@@ -5729,6 +5755,9 @@ It is initially set to false.</p>
 <!-- If we ever change this, e.g., add a ShadowRoot object constructor, that would have serious
      consequences for innerHTML. -->
 
+<p><a for=/>Shadow roots</a> have an associated <dfn for=ShadowRoot>slot assignment</dfn>
+("<code>manual</code>" or "<code>name</code>").</p>
```

Comment:
Did we bikeshed "name" vs. "named" at all here? Using "named" feels better to me for some reason, but no strong opinions.

Reply:
Yeah, since "manual" is an adjective (not a noun in the sense of a book of instructions), "named" would make it more consistent with it. I guess an alternative is to make "manual" a noun like "script" or "none".

I've also started to think that maybe we want to say "slotname" instead of just "name" since if we wanted to add, for example, the [ability to assign nodes based on element's local names](https://github.com/whatwg/html/issues/6051) then it would be regrettable to have "name" and then "elementname" / "localname".

Reply:
+1 for "named" (or "auto", but I guess we decided against that at some point in the past).

Reply:
Sounds like we're roughly agreed that "named" might be better. I'll modify...

Reply:
Done.